### PR TITLE
Irminger 9 and Papa 9a Glider updates

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
Deployment times/locations for Irminger gliders can be reviewed from the QLR attached here. Papa deployment time/location can be reviewed in QLR (3203-00905). Sensor configurations can be reviewed in the FCPs:

525: 3304-00160-00057
538: 3304-00160-00058
575: 3304-00170-00038
363: 3304-00170-00060

**Comment by cdobs** [comment]
[3202-00902-Quick-Look-Tables-Irminger-9-AR69-01.pdf](https://github.com/WHOIGit/ooicgsn-asset-management/files/9048240/3202-00902-Quick-Look-Tables-Irminger-9-AR69-01.pdf)

File: deployment/GI05MOAS-GL525_Deploy.csv [hunk 1]
```diff
@@ -2,4 +2,8 @@ CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentN
...
-AR35-05,,AJ20200506,,GI05MOAS-GL525-04-CTDGVM000,2,1,2019-08-06T17:23:00,2020-05-07T21:27:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-CTDGVM-09232,59.91594,-39.26567,,1000,1000,,,
\ No newline at end of file
+AR35-05,,AJ20200506,,GI05MOAS-GL525-04-CTDGVM000,2,1,2019-08-06T17:23:00,2020-05-07T21:27:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-CTDGVM-09232,59.91594,-39.26567,,1000,1000,,,
+AR69-01,,,,GI05MOAS-GL525-00-ENG000000,3,1,2022-06-28T16:58:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGCON-GLDROO-00525,59.81738,-39.11595,,1000,1000,,,
+AR69-01,,,,GI05MOAS-GL525-01-FLORDM000,3,1,2022-06-28T16:58:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-FLORDM-03579,59.81738,-39.11595,,1000,1000,,,
+AR69-01,,,,GI05MOAS-GL525-02-DOSTAM000,3,1,2022-06-28T16:58:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-DOSTAM-00569,59.81738,-39.11595,,1000,1000,,,
+AR69-01,,,,GI05MOAS-GL525-04-CTDGVM000,3,1,2022-06-28T16:58:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00525,CGINS-CTDGVM-09232,59.81738,-39.11595,,1000,1000,,,
```
Comment (rtravis-whoi) [inline deployment/GI05MOAS-GL525_Deploy.csv:9]: looks good

File: deployment/GI05MOAS-GL538_Deploy.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes,electrical.uid,assembly_template_revision
+AR69-01,,,,GI05MOAS-GL538-00-ENG000000,2,1,2022-06-28T17:27:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00538,CGCON-GLDROO-00538,59.81694,-39.11426,,1000,1000,,,
+AR69-01,,,,GI05MOAS-GL538-01-FLORDM000,2,1,2022-06-28T17:27:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00538,CGINS-FLORDM-03063,59.81694,-39.11426,,1000,1000,,,
+AR69-01,,,,GI05MOAS-GL538-02-DOSTAM000,2,1,2022-06-28T17:27:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00538,CGINS-DOSTAM-00436,59.81694,-39.11426,,1000,1000,,,
+AR69-01,,,,GI05MOAS-GL538-04-CTDGVM000,2,1,2022-06-28T17:27:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00538,CGINS-CTDGVM-00104,59.81694,-39.11426,,1000,1000,,,
```
Comment (rtravis-whoi) [inline deployment/GI05MOAS-GL538_Deploy.csv:5]: looks like this should be DOSTA-00438 according to the FCP

Comment (rtravis-whoi) [reply deployment/GI05MOAS-GL538_Deploy.csv:5]: fix to DOSTA SN good

**Review by reedan88 — APPROVED** [review]
The Papa 9a Glider 363 FCP document number is 3304-00160-00060.

File: deployment/GI05MOAS-PG575_Deploy.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+AR69-01,,AR69-01,,GI05MOAS-PG575-01-CTDGVM000,3,1,2022-06-28T17:24:00,2022-07-01T14:37:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-CTDGVM-09287,59.81703,-39.11459,,200,200,,,
+AR69-01,,AR69-01,,GI05MOAS-PG575-02-DOSTAM000,3,1,2022-06-28T17:24:00,2022-07-01T14:37:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-DOSTAM-00414,59.81703,-39.11459,,200,200,,,
+AR69-01,,AR69-01,,GI05MOAS-PG575-03-FLORTM000,3,1,2022-06-28T17:24:00,2022-07-01T14:37:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-FLORTM-03815,59.81703,-39.11459,,200,200,,,
+AR69-01,,AR69-01,,GI05MOAS-PG575-04-FLORTO000,3,1,2022-06-28T17:24:00,2022-07-01T14:37:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-FLORTO-01332,59.81703,-39.11459,,200,200,,,
+AR69-01,,AR69-01,,GI05MOAS-PG575-05-NUTNRM000,3,1,2022-06-28T17:24:00,2022-07-01T14:37:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-NUTNRM-00661,59.81703,-39.11459,,200,200,,,
+AR69-01,,AR69-01,,GI05MOAS-PG575-06-PARADM000,3,1,2022-06-28T17:24:00,2022-07-01T14:37:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00575,CGINS-PARADM-50200,59.81703,-39.11459,,200,200,,,
```
Comment (rtravis-whoi) [inline deployment/GI05MOAS-PG575_Deploy.csv:8]: recovery cruise added - looks good

**Review by rtravis-whoi — APPROVED** [review]

File: deployment/GI05MOAS-PG575_Deploy.csv (hunk 3 above)
Comment (reedan88) [inline deployment/GI05MOAS-PG575_Deploy.csv:8]: Looks good.

**Review by reedan88 — APPROVED** [review]
CUID added and looks good.

**Comment by rtravis-whoi** [comment]
[Irminger 9 AM Updates [DT-160]](https://trello.com/c/YNyU7qml/160-irminger-9-am-updates-dt-160)

**Pull request merged**